src/jest-reporters/eslint-check.js

```diff
@@ -0,0 +1,61 @@
+/* eslint-disable no-console */
+/* eslint-disable amo/only-tsx-files */
+/* eslint-disable @typescript-eslint/no-var-requires */
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
Note that I really should have been able to address this by adding an entry into `devDependencies` at https://github.com/mozilla/addons-code-manager/blob/master/.eslintrc#L16, but that doesn't work for some reason.